docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -114,7 +114,10 @@ The complete list of available targets is the following:
                <li><code>iosSimulatorArm64</code> — Apple iOS simulator on Apple Silicon platforms</li>
             </ul>
         </td>
-        <td>Requires a macOS host.</td>
+        <td>
+            <p>Requires a macOS host.</p>
```

Comment:
Maybe just "Requires a macOS host with Xcode and its command-line tools installed"?
Goes for other rows as well.